stat.proto

```diff
@@ -88,6 +88,7 @@ message StatJourneyResponse {
     optional string     last_pt_admin_id = 17;
     optional string     last_pt_admin_name = 18;
     optional string     last_pt_admin_insee = 19;
+    repeated string     tags = 20;
```

Comment:
Since there is a finite number of tags, maybe use an enum instead of a string ? 
It will be more efficient to read and write, and will results in smaller protobuf messages